cmake/Modules/CheckTypeSize.cmake

```diff
@@ -30,7 +30,7 @@
 # the macro undefined if the type does not exist.
 #
 # The variable "${VARIABLE}" may be "0" when CMAKE_OSX_ARCHITECTURES has
-# multiple architectures for building OS X universal binaries.  This
```

Comment:
Because this file is essentially copied from upstream cmake I think its best to leave it unpatched if thats OK?   It makes copariing with and updating from upstream easier if we minimize changes.